src/overlays/actors/ovl_En_Dnh/z_en_dnh.h

```diff
@@ -6,11 +6,22 @@
 struct EnDnh;
 
 typedef void (*EnDnhActionFunc)(struct EnDnh*, PlayState*);
+typedef s32 (*MsgEventFunc)(Actor*, PlayState*);
 
 typedef struct EnDnh {
     /* 0x000 */ Actor actor;
     /* 0x144 */ EnDnhActionFunc actionFunc;
-    /* 0x148 */ char unk_148[0x74];
+    /* 0x148 */ SkelAnime skelAnime;
+    /* 0x18C */ u16 unk18C;
+    /* 0x190 */ s32 msgEventScript;
+    /* 0x194 */ s32 unk194;
+    /* 0x198 */ s16 unk198;
+    /* 0x19A */ UNK_TYPE1 pad19A[0x2];
+    /* 0x19C */ s16 blinkTimer;
+    /* 0x19E */ s16 eyeTexIndex;
+    /* 0x1A0 */ MsgEventFunc msgEventCallback;
+    /* 0x1A4 */ Vec3s jointTable[2];
+    /* 0x1B0 */ Vec3s morphTable[2];
```

Comment:
```suggestion
    /* 0x1A4 */ Vec3s jointTable[OBJECT_TRO_LIMB_MAX];
    /* 0x1B0 */ Vec3s morphTable[OBJECT_TRO_LIMB_MAX];
```
You would then remove the `object_tro` include in the `.c` file, and instead include it here in the `.h` file.